darshan-util/pydarshan/darshan/tests/test_plot_exp_common.py

```diff
@@ -143,12 +144,16 @@
     ],
 )
 def test_xticks_and_labels(log_path, func, expected_xticklabels, mod):
+    if mod in ['H5F', 'H5D', 'PNETCDF_FILE', 'PNETCDF_VAR']:
+        pytest.xfail(reason="module not supported")
```

Comment:
Hmm, one thing I've mentioned to @shanedsnyder @carns is that I'd not like to see more capabilities added on the C analysis side--can we add the accumulators for these at the Python/`pandas` level moving forward? While it would be substantially easier to add on the C-side short-term I suspect, it would also dig us deeper in that side of things vs. the ultimate direction of interest (using common Python ecosystem tools for new capabilities/analyses).

Reply:
Also, do we not already have the expected values for some logs here, given that it was already working before? Wouldn't that make it easier to test newly-developed functionality? As far as I can tell, at least from the standpoint of the subset of plots we are working on now, we don't actually need any of the super-complicated C aggregator stuff anyway, so that seems like a decent starting point for the Python side agg stuff.